docs: redesign of aws integration guide (#2358)
kyma-project/telemetry-manager: docs/user/integration/aws-cloudwatch/README.md

   kubectl create secret generic aws-credentials -n $K8S_NAMESPACE --from-literal=AWS_ACCESS_KEY_ID={ACCESS_KEY} --from-literal=AWS_SECRET_ACCESS_KEY={SECRET_ACCESS_KEY} --from-literal=AWS_REGION={AWS_REGION}
   ```

## Deploy the custom collector

1. Export the Helm release name that you want to use. The release name must be unique for the chosen Namespace. Be aware that all resources in the cluster will be prefixed with that name. Run the following command:

    ```bash
    export HELM_OTEL_AWS_RELEASE="aws"
    ```

1. Update your Helm installation with the required Helm repository:

    ```bash
    helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
    helm repo update
    ```

1- Deploy the custom collector using Helm by calling:

```sh
helm upgrade --install -n $K8S_NAMESPACE $HELM_OTEL_AWS_RELEASE open-telemetry/opentelemetry-collector -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/aws-cloudwatch/values.yaml
```
<!-- markdown-link-check-disable -->

> The previous command uses the [values.yaml](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/aws-cloudwatch/values.yaml) provided in this `aws-cloudwatch` folder, which contains customized settings deviating from the default settings. The customizations in the provided `values.yaml` cover the following areas:

Reviewer: ```suggestion
The previous command uses the [values.yaml](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/aws-cloudwatch/values.yaml) provided in this `aws-cloudwatch` folder, which contains customized settings deviating from the default settings in the following ways:
```

<!-- markdown-link-check-enable -->

- Mount the values of Secret `aws-credentials` as environment variables
- Configure the OTEL exporter for logs assuming a LogGroup `/logs/kyma` and LogStream `default`

> - Configure the OTEL exporter for traces

Reviewer: ```suggestion
- Configure the OTel exporter for traces
```

   ```

2. Deploy a [TracePipeline](./../../03-traces.md):

   ```bash
   kubectl apply -f - <<EOF
   apiVersion: telemetry.kyma-project.io/v1alpha1
   kind: TracePipeline
   metadata:
     name: aws
   spec:
     output:
       otlp:
         endpoint:
           value: http://$HELM_OTEL_AWS_RELEASE-opentelemetry-collector.$K8S_NAMESPACE.svc.cluster.local:4317
   EOF
   ```

3. Deploy a [MetricPipeline](./../../04-metrics.md):

   ```bash
   kubectl apply -f - <<EOF
   apiVersion: telemetry.kyma-project.io/v1alpha1
   kind: MetricPipeline
   metadata:
     name: awsh
   spec:
     input:
       runtime:
         enabled: true
     output:
       otlp:
         endpoint:
           value: http://$HELM_OTEL_AWS_RELEASE-opentelemetry-collector.$K8S_NAMESPACE.svc.cluster.local:4317
   EOF
   ```

## Verify the Results

Verify that the logs, traces and metrics are exported to CloudWatch.

1. [Install the OpenTelemetry demo application](../opentelemetry-demo/README.md).
2. Go to `https://{AWS_REGION}.console.aws.amazon.com/cloudwatch`. Replace `{AWS_REGION}` with the region that you have chosen when [creating the Secret with AWS credentials](#create-a-secret-with-aws-credentials).

> 3. To verify the traces: under **Apllication Signals**, go to **Traces**.

Reviewer: ```suggestion
3. To verify the traces: under **Application Signals**, go to **Traces**.
```
Disregard this suggestion in case AWS truly uses "Apllication" for the UI element.

> 4. To verify the logs: under **Logs**, go to **Log groups** and select the log group and stream created as prerequisite and which is configured in the custom collector setup.

Reviewer: Are you sure it's "Log groups" on the UI? If you want to refer to UI elements, make sure to match the actual spelling on the UI.

Btw, this is why we [avoid click-level](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/80cf1399d10b47b482e833b6dbe9211c.html?locale=en-US&state=DRAFT&q=click-level) instructions; especially for someone else's (!) software.

   kubectl create secret generic aws-credentials -n $K8S_NAMESPACE --from-literal=AWS_ACCESS_KEY_ID={ACCESS_KEY} --from-literal=AWS_SECRET_ACCESS_KEY={SECRET_ACCESS_KEY} --from-literal=AWS_REGION={AWS_REGION}
   ```

## Deploy the Custom Collector

1. Export the Helm release name that you want to use. The release name must be unique for the chosen namespace. Be aware that all resources in the cluster will be prefixed with that name. Run the following command:

    ```bash
    export HELM_OTEL_AWS_RELEASE="aws"
    ```

1. Update your Helm installation with the required Helm repository:

    ```bash
    helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
    helm repo update
    ```

> 1- Deploy the custom collector using Helm by calling:

Author: ```suggestion
1. Deploy the custom collector using Helm by calling:
```
